src/Appwrite/Utopia/Database/Validator/Query/Select.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Query;
+
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+
+class Select extends Base
+{
+    protected array $schema = [];
+
+    /**
+     * Query constructor
+     *
+     */
+    public function __construct(array $attributes = [])
+    {
+        foreach ($attributes as $attribute) {
+            $this->schema[$attribute->getAttribute('key')] = $attribute->getArrayCopy();
+        }
+    }
+
+    /**
+     * Is valid.
+     *
+     * Returns true if method is TYPE_SELECT selections are valid
+     *
+     * Otherwise, returns false
+     *
+     * @param $query
+     * @return bool
+     */
+    public function isValid($query): bool
+    {
+        /* @var $query Query */
```

Comment:
can we do this here?
```suggestion
    public function isValid(Query $query): bool
    {
```

Reply:
Unfortunately not, this is the implementation of the abstract method from the base `Validator`